report/src/App.tsx

```diff
@@ -221,26 +236,29 @@ const App = () => {
           </div>
         </TableCell>
         {targetTotals.map((targetTotal) => {
-          const [supported, mismatch, stub, unsupported] = (
-            targetTotal as string
-          )
+          const [matching, mismatch, mock, missing] = (targetTotal as string)
             .split("/")
             .map((i) => parseInt(i as string));
 
-          const total = supported + mismatch + stub;
-          const percentage = pct(total, baselineCount as number);
+          const totalPresent = matching + mismatch + mock;
+          const total = totalPresent + missing;
+          const presentPct = pct(totalPresent, baselineCount as number);
+          const missingPct = pct(missing, total);
+          const mismatchPct = pct(mismatch, total);
+          const mockPct = pct(mock, total);
 
-          const tooltip = `Unsupported: ${unsupported}\nMismatch: ${mismatch}\nStubbed: ${stub}\nSupported: ${supported}`;
+          const tooltip = `Missing: ${missing}\nMismatch: ${mismatch}\nMocked: ${mock}\nMatching: ${matching}`;
 
           return (
             <TableCell>
               <div title={tooltip}>
                 <span className="text-sm font-semibold">
-                  {formatPct(percentage)}
+                  {formatPct(presentPct)}
                 </span>
                 <div className="text-xs">
                   <span>
-                    {unsupported}/{mismatch}/{stub}
+                    {formatPct(missingPct)}/{formatPct(mismatchPct)}/
+                    {formatPct(mockPct)}
                   </span>
```

Comment:
I think we should remove this - the amount of info in the table is overwhelming.

I'd rather use a proper tooltip to show the info on hover - using `title` has the drawback of introducing a small but anoying delay before the tooltip is displayed.

Quick Google search for better tooltips:
- https://github.com/cosbgn/tailwindcss-tooltips
- https://flowbite.com/docs/components/tooltips/
- https://www.material-tailwind.com/docs/html/tooltip
- https://pagedone.io/docs/tooltips

Reply:
I agree it's pretty overwhelming, but also I didn't realize there were tooltips until actually seeing them in the code. So discoverability takes a big hit with just that. And IMO these percentages are pretty important for full understanding.

Mind if we merge this as an iterative improvement and then try to figure out a better solution later?

Reply:
> I agree it's pretty overwhelming, but also I didn't realize there were tooltips until actually seeing them in the code.

Same here :)
Which is why a proprer tooltip would help.

But yes, we can merge and figure out later